aws/alarms/inputs.tf

```diff
@@ -39,11 +39,16 @@ variable "slack_webhook" {
   sensitive   = true
 }
 
-variable "sqs_deadletter_queue_arn" {
+variable "sqs_reliability_deadletter_queue_arn" {
   description = "ARN of the Reliability queue's SQS Dead Letter Queue"
   type        = string
 }
 
+variable "sqs_audit_log_deadletter_queue_arn" {
```

Comment:
I think this variable is not declared in the SQS output file

Reply:
It's on line 41 of the SQS output file.  I didn't need to add it as it was already there.